test/crdsvalidation/kongcacertificate/testcases/controlplaneref.go

```diff
@@ -119,5 +119,16 @@ var cpRef = testCasesGroup{
 			},
 			ExpectedUpdateErrorMessage: lo.ToPtr("spec.controlPlaneRef is immutable when an entity is already Programmed"),
 		},
+		{
+			Name: "no cpRef is valid",
```

Comment:
I'd consider adding a note here (and in other tests) that this will get reconciled by KIC when it uses types from this repo.

WDYT?

Reply:
Added comments.